uwp/uwp_main.h

```diff
@@ -70,6 +70,8 @@ namespace RetroArchUWP
       bool m_windowVisible;
       bool m_windowFocused;
       bool m_windowResized;
+      Windows::ApplicationModel::Activation::IActivatedEventArgs^ m_args;
```

Comment:
why are definitions for these functions created when they are only reused in one file can't these be defined within the main.cpp